README.md

```diff
@@ -66,6 +68,7 @@ Omnichannel offers an live chat widget (LCW) by default. You can use the Chat SD
 | OmnichannelChatSDK.getLiveChatConfig() | Get live chat config | |
 | OmnichannelChatSDK.getDataMaskingRules() | Get active data masking rules | |
 | OmnichannelChatSDK.getCurrentLiveChatContext() | Get current live chat context information to reconnect to the same chat | |
+| OmnichannelChatSDK.getChatReconnectContext() | Get current reconnectable chat context information to reconnect to the same chat | |
```

Comment:
`Get current reconnectable chat context information to reconnect to a previous existing chat session`